core/src/epicli/data/common/defaults/configuration/prometheus.yml

```diff
@@ -15,6 +15,7 @@ specification:
     - "--web.console.libraries=/etc/prometheus/console_libraries"                 # Directory should be the same as "config_directory"
     - "--web.console.templates=/etc/prometheus/consoles"                          # Directory should be the same as "config_directory"
     - "--web.listen-address=0.0.0.0:9090"                                         # Address that Prometheus console will be available
+    - "--web.enable-admin-api"                                                    # Enables administrative HTTP API
```

Comment:
We need to enable this permanently?

Reply:
Not really I guess. The problem is that doing this "on-demand" requires modification of the systemd unit and in result full stop / start cycle. I wanted to avoid restarts...